lib/req/steps.ex

```diff
@@ -818,7 +818,7 @@ defmodule Req.Steps do
   | identity      | Returns data as is                              |
 
   This step updates the following headers to reflect the changes:
-  - `content-legnth` is set to the length of the decompressed body
+  - `content-length` is set to the length of the decompressed body
```

Comment:
We'll have to add that `content-encoding` is removed, but this will be done in [another PR](https://github.com/wojtekmach/req/pull/190#pullrequestreview-1494951901). Hence the list with one item for now